accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceservice/AceService.java

```diff
@@ -62,13 +70,13 @@
 
     /** Common entry point for JMX and install hook.
      * 
-     * @param session
      * @param history
      * @param configurationFileContentsByFilename
      * @param authorizableInstallationHistorySet
+     * @param restrictedToPaths only apply ACLs to root paths as given
      * @throws Exception */
-    public void installConfigurationFiles(Session session, AcInstallationHistoryPojo history,
```

Comment:
Installation can take long - passing this session can mean it has to be open for a really long time (and then you have the concurrent session for hte authorizables). See https://github.com/Netcentric/accesscontroltool/issues/146 for details